lib/bigcommerce/prometheus/servers/thin/server.rb

```diff
@@ -31,6 +31,7 @@ def initialize(port: nil, host: nil, timeout: nil, logger: nil, thread_pool_size
             @rack_app = ::Bigcommerce::Prometheus::Servers::Thin::RackApp.new(timeout: timeout, logger: logger)
             super(@host, @port, @rack_app)
             ::Thin::Logging.logger = @logger
+            @logger.info "[bigcommerce-prometheus] Prometheus server started on #{@host}:#{@port}"
```

Comment:
Can you attach PoL?

I get that its only for debugging, but maybe we should also send a message to stderr as well?